styles.css

```diff
@@ -119,28 +127,7 @@ label {
 label img {
 	width: 50px;
 }
-/* https://www.freecodecamp.org/news/css-selectors-cheat-sheet/ */
+
 .lock:checked + label img {
 	content: url(assets/locked.png);
-}
```

Comment:
Removing the #box1, #box2 id that we aren't using is great, as it keeps the CSS short and readable.